s3torchbenchmarking/src/s3torchbenchmarking/datagen.py

```diff
@@ -129,13 +130,8 @@ def parse_human_readable_bytes(
 
     @staticmethod
     def write_dataset_config(disambiguator: str, dataset_cfg: Dict[str, Any]):
-        file_path = os.path.realpath(__file__)
-        cfg_path = (
-            Path(file_path).parent.parent.parent
-            / "conf"
-            / "dataset"
-            / f"{disambiguator}.yaml"
-        )
+        current_dir = os.getcwd()
```

Comment:
This only works if the application is launched in the right directory. The previous approach, even if more ugly, should work independantly